src/index.prod.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>GoldenSampleApp</title>
+  <base href="${BASE_HREF}">
```

Comment:
there's no need to do it this way, you can simply pass a base-href in the build options:

```
 "production": {
              "baseHref": "whatever",
```

Reply:
but how will that be reflected to the html when the user changes the env variable?

Reply:
If you put `"baseHref": "${BASE_HREF}",`, the resulting index.html would look exactly like the one you have now! The value would be replaced when the container starts.

Reply:
I'm probably misunderstanding something so just to clarify: where should I be putting this 
```
"production": { 
      "baseHref": "whatever",
```
?
I presumed angular.json; but that won't work I believe since it's not part of the statics that we're putting into the docker image.

Reply:
try putting that in the angular.json and running a build, then check the index.html in the dist folder

Reply:
Oh I see.. I thought you thought it was going to be changed in angular.json. Resolving now.